src/hooks/useAuthForm.tsx

```diff
@@ -18,17 +24,71 @@ export interface CommonFieldValues extends FieldValues {
   passwordConfirmation: string;
 }
 const useAuthForm = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    const storedUser = localStorage.getItem('user');
+    if (storedUser !== null) {
+      dispatch(setUser(JSON.parse(storedUser)));
+    }
+  }, []);
+  const navigate = useNavigate();
   const {
     handleSubmit,
     register,
     formState: { errors },
   } = useForm<CommonFieldValues>();
 
   const onSignInSubmit: SubmitHandler<ISignIn> = (data: ISignIn) => {
-    console.log('Form data:', data);
+    axios
```

Comment:
Should be in asyncThunk, please read the doc

Reply:
asyncThunk added